tests/integration/suite/sentry/validator/kubernetes/standalone.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubernetes
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/x509"
+	"encoding/pem"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/dapr/dapr/pkg/modes"
+	sentrypbv1 "github.com/dapr/dapr/pkg/proto/sentry/v1"
+	"github.com/dapr/dapr/pkg/sentry/server/ca"
+	"github.com/dapr/dapr/tests/integration/framework"
+	"github.com/dapr/dapr/tests/integration/framework/process/sentry"
+	"github.com/dapr/dapr/tests/integration/suite"
+)
+
+func init() {
+	suite.Register(new(standalone))
+}
+
+// standalone tests Sentry in standalone mode in kubernetes
+type standalone struct {
+	sentry *sentry.Sentry
+}
+
+func (k *standalone) Setup(t *testing.T) []framework.Option {
+	rootKey, err := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
+	require.NoError(t, err)
+	bundle, err := ca.GenerateBundle(rootKey, "integration.test.dapr.io", time.Second*5, nil)
+	require.NoError(t, err)
+
+	kubeAPI := kubeAPI(t, kubeAPIOptions{
+		bundle:         bundle,
+		namespace:      "mynamespace",
+		serviceAccount: "myserviceaccount",
+		appID:          "myappid",
+	})
+
+	k.sentry = sentry.New(t,
+		sentry.WithNamespace("sentrynamespace"),
+		sentry.WithMode(string(modes.StandaloneMode)),
+		sentry.WithCABundle(bundle),
+		sentry.WithTrustDomain("integration.test.dapr.io"),
+	)
+
+	return []framework.Option{
```

Comment:
This is a nit, but can this test go to a `tests/integration/suite/sentry/validator/standalone/<filename>.go` path? bc as is, its a `standalone.go` file inside the `kubernetes` directory which is a bit confusing

Reply:
ofc! That's def a better place for it!

Reply:
thank you